### DISCUSSION TITLE
ESP8285 2Mb no Matter in Configuration, no Google Home

### DISCUSSION THREAD

**Discussion by jenom1957** [arendst/Tasmota#22078] [Q&A] [answered]
Installed Tasmota v14.2.0 on a Esp-02S ESP8255 , 2 Mb memory, mini switch.
However, there is no Matter in Configuration, which supposed to be needed to get Google Home working.
Any suggestion? Thank You!

**Comment by s-hadinger** [comment, marked as answer]
The chip is too limited to support Matter, especially RAM is too tiny. 

However you can still use an additional esp32 to act as a bridge and allow Google Home to control it via the bridge. 

Check the documentation

**Comment by jenom1957** [comment]
It works in Tasmota with Amazon's Alexa as a Belkin simulation.
The original firmware used Smart Life or Tuya Smart and I think it supposed to work with Google.
Is there any way get it working with Google & Tasmota ?
Thanks!

**Reply by sfromis** [reply]
Not clear what you mean by "it", but with total firmware replacement it does not matter (sic) what features the old firmware had or not had. And to be clear, Tasmota cannot integrate with Tuya apps, as getting rid of such things is a major reason to replace the Tuya firmware on devices.

But you already know the two paths to getting it to work with Tasmota, bridging via ESP32, or emulation of older protocols.

**Comment by jenom1957** [comment]
"IT" = the actual device, DIY Smart Switch 
So is there any way to make this switch (Tasmota firmware) to work with Google Home ?
Thanks!

**Reply by s-hadinger** [reply]
Yes, as mentioned with the help of an external ESP32 to act as a bridge - see "Add remote Tasmota"

https://tasmota.github.io/docs/Matter/#add-remote-tasmota-or-openbk

**Reply by s-hadinger** [reply]
Also keep in mind that Matter requires a hardware gateway in your home (Nest Mini...), not just the app on your phone.

**Comment by mhalano** [comment]
I think the Matter doesn't work in this chip because of the lack of Bluetooth, different from ESP32-C2 let's say because, IMHO, you need Bluetooth to commission the device. Also lack of RAM is also a good reason.

**Reply by sfromis** [reply]
AFAIK, the Matter implementation in Tasmota does not rely on Bluetooth, but does need some space, meaning that the chopped variant with measly 2MB of flash is not a good option to hope to use.

**Reply by s-hadinger** [reply]
Ram is too low for Matter to run of this chip
